src/Bridges/Constraint/bridges/soc_rsoc.jl

```diff
@@ -13,7 +13,7 @@
 
 ## Assumptions
 
-  * `SOCtoRSOCBridge` assumes that ``|x| \\ge 1``.
+  * `SOCtoRSOCBridge` assumes that the dimension of the cone is at least one.
```

Comment:
```suggestion
  * `SOCtoRSOCBridge` assumes that the length of `x` is at least one.
```

I don't think it works if you have `[t] in SecondOrderCone(1)`.

Reply:
Ah yes, I meant strictly greater